modules/base/base/module.py

```diff
@@ -598,6 +598,10 @@ async def _bookmark(
             icon_url=message.author.display_avatar.replace(size=64).url,
         )
 
+        embed.add_field(
+            name=_(utx, "Author ID"),
+            value=_(utx, "{id}").format(id=message.author.id),
```

Comment:
This is not necessary, as this message is just a number with no other text; `value=message.author.id` will do the same.

Reply:
That's true, adjusted.